src/libaktualizr/package_manager/packagemanagerfake_test.cc

```diff
@@ -29,6 +29,7 @@ TEST(PackageManagerFake, FinalizeAfterReboot) {
   Uptane::Target target("pkg", {Uptane::Hash(Uptane::Hash::Type::kSha256, "hash")}, 1, "");
   auto result = fakepm.install(target);
   EXPECT_EQ(result.result_code, data::ResultCode::Numeric::kNeedCompletion);
+  storage->savePrimaryInstalledVersion(target, InstalledVersionUpdateMode::kPending);
```

Comment:
I know this is out of scope of this PR, just noticed that savePrimaryInstalledVersion() neither returns error code nor throws exception in case of failure.

Reply:
Yes, that's one of the places that I've mentioned this morning. Feel free to fix it here, of course, or you can wait for me to prepare a PR with few more similar fixes.